### ISSUE TITLE
Image Flip in Render

### ISSUE THREAD

**Issue by sweisernovak** [NeuroMorph-EPFL/NeuroMorph#5]
![image](https://user-images.githubusercontent.com/1775977/56918544-2e2d1300-6a73-11e9-9f9f-7b71b75030e9.png)

My image is flipped in the Render view relative to the Viewport. 

Any ideas for troubleshooting?

Thanks!
Sam

**Comment by sweisernovak** [comment]
SOLVED. When importing the image ladder, the slice was X-rotated by 180° and offset in Y by the stack length (Y dimension). I had corrected for that after I figured it out - didn't try rendering until I had already sorted out my desired animation sequence etc. Reapplying the preloaded correction (ie., 180° in delta-X-rot and 1.92 in delta-Y offset) (delta so that it propagated to all keyframes) to my animation caused the Viewport to appear misaligned but in the render, the model lines up with the slices again. OK! Thanks for this wonderful tool. S
